app/pages/external/ApprovedOperatorPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.external
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object ApprovedOperatorPage extends QuestionPage[Boolean] {
```

Comment:
With the external pages I think we usually make them read only e.g. ReadOnlyPage[Boolean]